app/renderer/components/common/dialog.js

```diff
@@ -40,6 +43,7 @@ class Dialog extends ImmutableComponent {
   render () {
     return <div className={cx({
       dialog: true,
```

Comment:
is it needed somewhere? we are removing style class and I can't see any test using it.